### ISSUE TITLE
empty sandbox deposition lists - Can not set embargo date for Zenodo Sandbox deposit

### ISSUE THREAD

**Issue by rkrug** [ropenscilabs/deposits#93]
I have a dataset I want to publish as embargoed - so far so good. But I can not set the time until when it is embargoed.

```
r$> cli
<deposits client>
 deposits service : zenodo
           sandbox: TRUE
         url_base : https://sandbox.zenodo.org/api/
 Current deposits : 1 (see 'deposits' element for details)

   hostdata : <none>
   metadata : 8 terms (see 'metadata' element for details)

r$> cli$metadata$accessRights
[1] "embargoed"

r$> cli$deposit_embargo("2024-01-01")
Error: Client not associated with any deposit which can be embargoed. Please first use `deposit_new()` or `deposit_retrieve()` methods.
```

**Comment by rkrug** [comment]
Happy to provide more details if needed.

**Comment by mpadge** [comment]
Likely because of https://github.com/ropenscilabs/deposits/issues/92#issuecomment-1841174815:
> The deposit_id value should be populated from the Zenodo sandbox data, but if that's all empty, there's no value to use.

This issue can serve as a placeholder to address all issues related to empty sandbox deposition lists.

**Comment by rkrug** [comment]
Have you tried if it works on Zenodo non-sandbox? 

I am asking because I have to make some deposits in December and would like to use `deposits` but have to do it manually if it does not work on Zenodo non-sandbox.

**Comment by mpadge** [comment]
Yep, on non-sandbox everything seems to work as expected. The sandbox is, however, a complete mess. The web interface dumps the explicit validation errors at the top, which is useful, but also shows a bunch of errors some of which are simply impossible to resolve. I have no idea what is going on there, and will have to allocate some time next week to check it all out. Sorry for the inconvenience, but at least the non-sandbox seems to be functioning same as ever.
